frontend/ui/widget/inputdialog.lua

```diff
@@ -128,6 +128,8 @@ local InputDialog = InputContainer:new{
     input_type = nil,
     deny_keyboard_hiding = false, -- don't hide keyboard on tap outside
     enter_callback = nil,
+    strike_callback = nil, -- call this on every keystroke (used in KOTerm: TermInputText)
+    inputtext_class = InputText, -- (KOTerm provides TermInputText)
```

Comment:
```suggestion
    strike_callback = nil, -- call this on every keystroke (used by Terminal plugin's TermInputText)
    inputtext_class = InputText, -- (Terminal plugin provides TermInputText)
```

Reply:
This should be the last trace of KOTerm -> farewell :(

Reply:
(We'll still be calling it KOTerm, between us :)